io.github.yairm210.unciv.appdata.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.yairm210.unciv</id>
+  <name>UnCiv</name>
+  <url type="homepage">https://github.com/yairm210/Unciv/</url>
+  <url type="bugtracker">https://github.com/yairm210/UnCiv/issues</url>
+  <url type="donation">https://patreon.com/yairm210</url>
+  <url type="translate">https://github.com/yairm210/Unciv/wiki/Translating</url>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MPL-2.0</project_license>
+  <developer_name>Yair Morgenstern</developer_name>
+  <summary>Turn-based strategy game</summary>
+  <summary xml:lang="fr">Jeu de stratégie au tour par tour</summary>
+
+  <description>
+    <p>A reimplementation of the most famous civilization-building game ever—fast, small, no ads, free forever!</p>
+    <p>Build your civilization, research technologies, expand your cities and defeat your foes!</p>
+    <p>Requests? Bugs? Visit the todo list at https://github.com/yairm210/UnCiv/issues, every small help is welcome!</p>
+    <p>Questions? Comments? Just bored? Join us on https://discord.gg/bjrB4Xw</p>
+    <p>Want to help translating the game into your language? Send me an email to yairm210@hotmail.com.</p>
+    <p>Grok Java, Kotlin or C#? Join us at https://github.com/yairm210/UnCiv.</p>
+    <p>The world awaits! Will you build your civilization into an empire that will stand the test of time?</p>
+    <p>Internet permissions required in order to download large font files, especially for Chinese, which, if included by default, would more than double (!) the app size.</p>
+  </description>
+
+  <launchable type="desktop-id">io.github.yairm210.unciv.desktop</launchable>
+
+  <categories>
+    <category>Game</category>
+    <category>2DGraphics</category>
+    <category>StrategyGame</category>
+    <category>Java</category>
+  </categories>
```

Comment:
The categories are already set in the desktop file. They are merged together automatically

Reply:
Should I only leave them in the desktop file, then? Is it flathub-specific, BTW?

Reply:
Nop, the metainfo and the desktop file are merged together using appstream-compose. And yes, just in the desktop file is enough.